AK/ByteBuffer.h

```diff
@@ -217,7 +212,7 @@ class ByteBuffer {
         other.m_inline = true;
     }
 
-    void internal_trim(size_t size, bool may_discard_existing_data)
+    void trim(size_t size, bool may_discard_existing_data)
```

Comment:
This `may_discard_existing_data` argument seems to be ignored, peeking ahead tells me that the next commit uses it.
Does ignoring it not cause issues?

Reply:
Seems like I was blind.